src/pint/models/pulsar_binary.py

```diff
@@ -185,7 +187,33 @@ def apply_units(self):
             bparObj.value = bparObj.value * u.Unit(bparObj.units)
 
     def update_binary_object(self, toas=None, acc_delay=None):
-        """Update binary object instance for this set of parameters/toas."""
+        """Update stand alone binary's parameters and toas from PINT-facing object.
+
+        This function passes the PINT-facing object's parameter values and TOAs
+        to the stand-alone binary object. If the TOAs are not provided, it only
+        updates the parameters not the TOAs.
+
+        Parameters
+        ----------
+        toas: `~pint.toa.TOAs` object, optional
```

Comment:
The backticks and tilde do not work in this context. Just giving the class name, undecorated, should be correctly interpreted by Sphinx.